README.md

```diff
@@ -60,6 +60,7 @@ Below are some of the developer ambassador programs in alphabetical order (if yo
 * [Microsoft MVP](https://mvp.microsoft.com/en-US/mvp) - The Microsoft MVP Program connects technical community leaders with Microsoft to promote engagement, advocacy, and knowledge sharing on Microsoft Products & Services.
 * [Microsoft Student Ambassador](https://mvp.microsoft.com/en-US/studentambassadors) - Students who are passionate about building AI-driven solutions with Microsoft technology. Accelerate innovation and grow the skills you need to have greater impact in the projects and communities that matter to you. * For students only.
 * [MongoDB Community Advocacy](https://www.mongodb.com/community/advocacy/) - MongoDB Community Advocacy Program is a global community for passionate and dedicated MongoDB advocates. The program enables enthusiastic program members to grow their knowledge, profile and leadership through engagement and advocacy of all users of MongoDB technologies. It has champions and creators subdivisions.
+* [Notion Ambassadors](https://www.notion.so/notion/Notion-Ambassadors-40c3b3ee8c744e7fad34ab4ac9765773) - Notion Ambassadors are passionate, volunteer enthusiasts who teach and share Notion with others. They lead local communities, organize online and offline events, produce courses and tutorials, create templates, share translations, and much more.
```

Comment:
They even have Notion Campus leaders: https://www.notion.so/notion/Notion-Campus-Leaders-Program-5817b00cbaa244bca9e0e498804cbab4 

But, I think it is a program open to anyone, you don't need to be a "developer" or technical. I will think about adding it if you add the Campus leader one too. Thanks @LaxmanMaharjan .

Reply:
Yeah, True about the program being open to anyone.

Notion Campus Leaders is cool too, let me that as well.

Reply:
It should be helpful to people, adding it in. Thanks for contributing @LaxmanMaharjan .